src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/idam/IdamTestService.java

```diff
@@ -96,6 +100,16 @@ public User createDefendant(final String letterHolderId) {
         return userService.authenticateUser(email, password);
     }
 
+    private void createUser(CreateUserRequest createUserRequest) {
+        idamTestApi.createUser(createUserRequest);
+        //recommended delay from SIDAM team to stop intermittent auth failures
+        try {
+            Thread.sleep(1000);
```

Comment:
So we still need this delay. IMHO This should be a temporary solution not permanent.  Should we create a story/bug for IDAM? and have a tech debit story in our Jira to link with that?